test/fuzz/oss_fuzz_build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -eu
+# Copyright 2024 The Sigstore Authors.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+go get github.com/AdamKorcz/go-118-fuzz-build/testing
```

Comment:
Is there any plan to have this in oss-fuzz?

Reply:
Yes, but it will take a few months.